framework/moose.mk

```diff
@@ -72,7 +82,7 @@ $(1):
 	@$$(shell mkdir -p $$@)
 endef
 
-include_files	:= $(shell find $(FRAMEWORK_DIR)/include -regex "[^\#~]*\.h")
+include_files	:= $(shell find $(FRAMEWORK_DIR)/include \( -regex "[^\#~]*\.h" ! -name "*MooseConfig.h" \))
```

Comment:
Hopefully we don't run into weird platform differences in find. I really hate find.